cluster/gce/trusty/configure-helper.sh

```diff
@@ -612,6 +613,21 @@ start_kube_apiserver() {
     params="${params} --etcd-quorum-read=${ETCD_QUORUM_READ}"
   fi
 
+  if [[ "${ENABLE_APISERVER_BASIC_AUDIT:-}" == "true" ]]; then
+    # We currently only support enabling with a fixed path and with built-in log
+    # rotation "disabled" (large value) so it behaves like kube-apiserver.log.
+    # External log rotation should be set up the same as for kube-apiserver.log.
+    params="${params} --audit-log-path=/var/log/kube-apiserver-audit.log"
```

Comment:
is it wrong that I like the params `+=` style you used the other 2 times. And is sometimes used in this file?